spec/conversions.md

```diff
@@ -50,8 +50,8 @@ The implicit numeric conversions are:
 *  From `byte` to `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`.
 *  From `short` to `int`, `long`, `float`, `double`, or `decimal`.
 *  From `ushort` to `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`.
-*  From `int` to `long`, `float`, `double`, or `decimal`.
-*  From `uint` to `long`, `ulong`, `float`, `double`, or `decimal`.
+*  From `int` to `nint`, `long`, `float`, `double`, or `decimal`.
+*  From `uint` to `nuint`, `long`, `ulong`, `float`, `double`, or `decimal`.
```

Comment:
These updates made due to the fact that the following compiles without error:

```
int x = 5;
nint y = x;

uint a = 5;
nuint b = a;
```

Correct me if that's not correct.